packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -1,15 +1,49 @@
 import CozyClient, { Q } from 'cozy-client'
 
-const DEFAULT_CACHE_TIMEOUT_QUERIES = 9 * 60 * 1000
+const DEFAULT_CACHE_TIMEOUT_QUERIES = 10 * 60 * 1000
 const defaultFetchPolicy = CozyClient.fetchPolicies.olderThan(
   DEFAULT_CACHE_TIMEOUT_QUERIES
 )
 
-export const buildKonnectorQuery = slug => ({
-  definition: () =>
-    Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+export const buildKonnectorQuery = (slug, source) => {
+  const query = {
+    definition: () =>
+      Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+    options: {
+      as: `konnector-${slug}-${source}`,
+      fetchPolicy: defaultFetchPolicy
+    }
+  }
+  const queryDef = query.definition()
+  queryDef.sources = [source]
+
+  return { definition: queryDef, options: query.options }
+}
+
+export const buildIsAccountConnectedQuery = sourceAccount => ({
+  definition: Q('io.cozy.accounts').getById(sourceAccount),
+  options: {
+    as: `isAccountConnected-${sourceAccount}`,
+    fetchPolicy: defaultFetchPolicy
+  }
+})
+
+export const buildIsInErrorTriggersQuery = (slug, sourceAccount) => ({
```

Comment:
Same logic as before, "isInError" can be removed since it has nothing specific with "errors".

I propose to rename the arguments into konnectorSlug, accountId, to make it clear that sourceAccount is not an object.